### PR TITLE
Added editorconfig file

### PR DESCRIPTION
**Pull request by Gumbees**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = unset
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.js]
+charset = utf-8
```
Comment (egyptianbman) [inline .editorconfig]: You can probably move this to the `[*]` block.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# 4 space indentation
+[*.{js,json}]
+indent_style = space
```
Comment (egyptianbman) [inline .editorconfig]: Same with this. :)

**Review by egyptianbman — CHANGES_REQUESTED** [review]

**Comment by Gumbees** [comment]
@egyptianbman updated.

**Review by egyptianbman — APPROVED** [review]
Thanks!

**Comment by egyptianbman** [comment]
We can probably add the same file to the other repos.

**Pull request merged**